### ISSUE TITLE
Any browsers build for AWS Lambda?

### ISSUE THREAD

**Issue by und3fined** [microsoft/playwright#626]
Hi MS,

Team can opensource scirpt build brower for lambda or a version for lambda?

**Comment by XPilot** [comment]
It would be nice to have browser binaries that could be deployed for AWS.

The chromium browser binary can be found here: https://github.com/alixaxel/chrome-aws-lambda which allows the usage of puppeteer, could not find any reference for Firefox, Edge or Safari.

**Comment by und3fined** [comment]
@XPilot currently, i'm use a fork of puppeteer and our chrome build for lambda
I having some issues with launching chrome on lambda, i want to experiment with firefox and webkit

**Comment by aslushnikov** [comment]
> Team can opensource scirpt build brower for lambda or a version for lambda?

@und3fined We don't have it!

However, all our infrastructure for browser building lives under `//browser_patches` - so should be doable externally for both firefox and webkit. (cc'ing @alixaxel in case this is something they're interested in)

**Comment by alixaxel** [comment]
@XPilot I did attempt to get Firefox headless working for it a while ago, but back then it required xvfb.

I might try again soon if something changed. As for WebKit, I will also have a look.

**Comment by aslushnikov** [comment]
@alixaxel note: we also have webkit-wpe builds (e.g. [minibrowser-wpe r1120](https://playwright.azureedge.net/builds/webkit/1120/minibrowser-wpe.zip)) that don't require xvfb

**Comment by austinkelleher** [comment]
I'm actively working on https://github.com/austinkelleher/playwright-aws-lambda.

I was able to get this working on Lambda with `browser.defaultContext()` (which has been removed in v0.11.0). Current hack is to use `browser._defaultContext`. Calling `newPage` with a new context fails with `Error: Protocol error (Target.createTarget): Target closed.`. For what it's worth, I'm only using Chrome 80.0.3987.0.

Any thoughts @aslushnikov? Really great work on Playwright so far! Very exciting.
